.changelog/README.md

```diff
@@ -0,0 +1,132 @@
+# How To Use
+
+Nomad uses [`go-changelog`](https://github.com/hashicorp/go-changelog) to generate its changelog on release.
+
+To install, run the following command:
+
+```
+go get github.com/hashicorp/go-changelog/cmd/changelog-build
```

Comment:
```suggestion
go install github.com/hashicorp/go-changelog/cmd/changelog-build@latest
```

We should encourage folks to use `go install` instead: https://github.com/golang/go/issues/43684

Reply:
Good point -- thanks!